.github/workflows/update_package_properties.yml

```diff
@@ -30,17 +30,23 @@ jobs:
         run: sudo apt install libcurl4-openssl-dev libssl-dev
 
       - name: Clone tools branch
-        run: git clone -b v0.8.0 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b issue-149 --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
Should be updated with a tag before merging

Reply:
ping

Reply:
After all tools PR's finished we can tag and change it

Reply:
Fixed